<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditorDialog</class>
 <widget class="QDialog" name="ItemEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Тип работы</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabJobType">
      <attribute name="title">
       <string>Тип работы</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="10" column="0">
        <widget class="QLabel" name="lblActionDateChanger">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Модификатор &amp;даты действия</string>
         </property>
         <property name="buddy">
          <cstring>cmbActionDateChanger</cstring>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QComboBox" name="cmbActionDateChanger">
         <item>
          <property name="text">
           <string>Не меняется</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Дата окончания</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="edtCode"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblRegionalCode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Региональный код</string>
         </property>
         <property name="buddy">
          <cstring>edtRegionalCode</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="edtRegionalCode"/>
       </item>
       <item row="7" column="1">
        <widget class="CActionStatusComboBox" name="cmbActionStatusChanger"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lblActionPersonChanger">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Модификатор &amp;исполнителя действия</string>
         </property>
         <property name="buddy">
          <cstring>cmbActionPersonChanger</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="cmbActionPersonChanger">
         <item>
          <property name="text">
           <string>Не меняется</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Пользователь</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Назначивший действие</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ответственный за событие</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="lblNote">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Примечание</string>
         </property>
         <property name="buddy">
          <cstring>edtNote</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lblActionStatusChanger">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Модификатор &amp;статуса действия</string>
         </property>
         <property name="buddy">
          <cstring>cmbActionStatusChanger</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="edtName"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblCode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Код</string>
         </property>
         <property name="buddy">
          <cstring>edtCode</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Наименование</string>
         </property>
         <property name="buddy">
          <cstring>edtName</cstring>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="15" column="1" rowspan="2">
        <widget class="QTextEdit" name="edtNote"/>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="edtCapacity">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Для групповых действий: максимальный размер группы&lt;/p&gt;&lt;p&gt;Для прочих: 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblCapacity">
         <property name="text">
          <string>&amp;Ёмкость</string>
         </property>
         <property name="buddy">
          <cstring>edtCapacity</cstring>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="lblListContext">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Контекст &amp;печати списка</string>
         </property>
         <property name="buddy">
          <cstring>edtListContext</cstring>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLineEdit" name="edtListContext"/>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="lblTicketDuartion">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Длительность номерка</string>
         </property>
         <property name="buddy">
          <cstring>edtTicketDuration</cstring>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QSpinBox" name="edtTicketDuration">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Длительность выполнения работы, в днях&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lblManualExecutionAssignments">
         <property name="text">
          <string>Предлагать ручное выполнение назначений</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QCheckBox" name="chkManualExecutionAssignments">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabActionTypes">
      <attribute name="title">
       <string>Действия</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="CInDocTableView" name="tblJobTypeActions"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
  <customwidget>
   <class>CActionStatusComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">Events/ActionStatus.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>edtCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>edtRegionalCode</tabstop>
  <tabstop>edtCapacity</tabstop>
  <tabstop>cmbActionStatusChanger</tabstop>
  <tabstop>chkManualExecutionAssignments</tabstop>
  <tabstop>cmbActionPersonChanger</tabstop>
  <tabstop>cmbActionDateChanger</tabstop>
  <tabstop>edtTicketDuration</tabstop>
  <tabstop>edtListContext</tabstop>
  <tabstop>edtNote</tabstop>
  <tabstop>tblJobTypeActions</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
